pkgs/by-name/sk/skyscraper/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  qt5,
+  p7zip,
+
+  # Whether to compile with XDG support
+  # (See: https://gemba.github.io/skyscraper/XDG/)
+  enableXdg ? false,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "skyscraper";
+  version = "3.14.0";
+
+  src = fetchFromGitHub {
+    owner = "Gemba";
+    repo = "skyscraper";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-UgPo9qEsYYYl2RgdPtPmToAruPKuvJ3FSgVrXa94kdA=";
+  };
+
+  strictDeps = true;
+
+  nativeBuildInputs = [
+    qt5.wrapQtAppsHook
+    qt5.qmake
+  ];
+
+  postPatch = lib.optionalString enableXdg ''
+    substituteInPlace skyscraper.pro --replace-fail "#DEFINES+=XDG" "DEFINES+=XDG"
+  '';
+
+  postFixup = ''
+    wrapProgram $out/bin/Skyscraper \
+      --prefix PATH : ${lib.makeBinPath [ p7zip ]}
+  '';
```

```suggestion
  preFixup = ''
    qtWrapperArgs+=(--prefix PATH : ${lib.makeBinPath [ p7zip ]})
  '';
```
Since it gets double wrapped with Qt otherwise

---

Done.